community/bitcoin/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=bitcoin
-pkgver=0.17.1
+pkgver=0.18.1
 _ver=${pkgver/_/}
 pkgrel=2
```

Comment:
when upgrading a package reset pkgrel to 0

Reply:
Thanks for the prompt response, didn't thought it would be as fast!
Fixed!